dts/bindings/fs/zephyr,fstab,littlefs.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Description of pre-defined file systems.
+
+compatible: "zephyr,fstab,littlefs"
+
+include: "zephyr,fstab-base.yaml"
+
+properties:
+  # num-files and num-dirs are not filesystem-specific.
+
+  read-size:
```

Comment:
I wonder if these should be `littlefs,read-size` or some prefix like that to make clear the `littlefs` specific properties.

Reply:
Eh.  I guess it could, but it's within a compatible that clearly identifies it as littlefs.  I'd rather keep things short because using the DT macros in code is always a battle against the 80-character line length limit.

Reply:
I think that fs specific options are fine; as far as I understand the fs driver is always responsible for interpretation of it's own options in DTS and it will not touch anything that is not known to it prior.